[2024-12-17T16:41:17.737+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-12-17T16:41:17.748+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: PRCS_TEST_FRAMEWORK_DEPN_2.Run_bash_to_python_file manual__2024-12-17T16:41:16.720879+00:00 [queued]>
[2024-12-17T16:41:17.751+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: PRCS_TEST_FRAMEWORK_DEPN_2.Run_bash_to_python_file manual__2024-12-17T16:41:16.720879+00:00 [queued]>
[2024-12-17T16:41:17.752+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 6
[2024-12-17T16:41:17.759+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): Run_bash_to_python_file> on 2024-12-17 16:41:16.720879+00:00
[2024-12-17T16:41:17.764+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=1690) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-12-17T16:41:17.765+0000] {standard_task_runner.py:72} INFO - Started process 1703 to run task
[2024-12-17T16:41:17.765+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'PRCS_TEST_FRAMEWORK_DEPN_2', 'Run_bash_to_python_file', 'manual__2024-12-17T16:41:16.720879+00:00', '--job-id', '2148', '--raw', '--subdir', 'DAGS_FOLDER/script/PRCS_TEST_FRAMEWORK_DEPN_2.py', '--cfg-path', '/tmp/tmp0lz290m9']
[2024-12-17T16:41:17.766+0000] {standard_task_runner.py:105} INFO - Job 2148: Subtask Run_bash_to_python_file
[2024-12-17T16:41:17.793+0000] {task_command.py:467} INFO - Running <TaskInstance: PRCS_TEST_FRAMEWORK_DEPN_2.Run_bash_to_python_file manual__2024-12-17T16:41:16.720879+00:00 [running]> on host 2dca8d069141
[2024-12-17T16:41:17.834+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='test-framework' AIRFLOW_CTX_DAG_ID='PRCS_TEST_FRAMEWORK_DEPN_2' AIRFLOW_CTX_TASK_ID='Run_bash_to_python_file' AIRFLOW_CTX_EXECUTION_DATE='2024-12-17T16:41:16.720879+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-12-17T16:41:16.720879+00:00'
[2024-12-17T16:41:17.835+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-12-17T16:41:17.840+0000] {logging_mixin.py:190} INFO - hi there3 2024-12-17 16:41:17.840168
[2024-12-17T16:41:17.841+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-12-17T16:41:17.846+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-12-17T16:41:17.847+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=PRCS_TEST_FRAMEWORK_DEPN_2, task_id=Run_bash_to_python_file, run_id=manual__2024-12-17T16:41:16.720879+00:00, execution_date=20241217T164116, start_date=20241217T164117, end_date=20241217T164117
[2024-12-17T16:41:17.865+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-12-17T16:41:17.878+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-12-17T16:41:17.880+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
